libnopegl/src/utils.h

```diff
@@ -158,4 +162,20 @@ char *ngli_numbered_lines(const char *s);
 int ngli_config_copy(struct ngl_config *dst, const struct ngl_config *src);
 void ngli_config_reset(struct ngl_config *config);
 
+static inline uint32_t ngli_clz(uint32_t x)
+{
+#ifdef _MSC_VER
+    unsigned long ret;
+    _BitScanReverse(&ret, x);
+    return 31 - ret;
+#else
+    return __builtin_clz(x);
```

Comment:
can you add a mandatory builtin check in `libnopegl/meson.build`?

Reply:
Done.